### PR TITLE
Alternate URL resource

### PR DESCRIPTION
**Pull request by BrianFarnhill**
Addresses issue #44

### CODE REVIEW

**Comment by BrianFarnhill** [comment]
@camiloborges @caadam @anlynes First one to code review gets high fives

File: Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.psm1 [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        [parameter(Mandatory = $true)]  [System.String] $WebAppUrl,
+        [parameter(Mandatory = $true)]  [ValidateSet("Default","Intranet","Extranet","Custom","Internet")] [System.String] $Zone,
+        [parameter(Mandatory = $true)]  [System.String] $Url,
+        [parameter(Mandatory = $false)] [System.Management.Automation.PSCredential] $InstallAccount
+
+    )
+
+    $CurrentValues = Get-TargetResource @PSBoundParameters
+    Write-Verbose -Message "Testing alternate URL configuration"
+    return Test-xSharePointSpecificParameters -CurrentValues $CurrentValues -DesiredValues $PSBoundParameters -ValuesToCheck @("Url")
```
Comment (camiloborges) [inline Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.psm1]: Do you need to test zone and webappurl?

File: Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.schema.mof [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+*/
+
+[ClassVersion("1.0.0.0"), FriendlyName("xSPAlternateUrl")]
+class MSFT_xSPAlternateUrl : OMI_BaseResource
+{
+    [Key] String WebAppUrl;
+    [Key, ValueMap{"Default","Intranet","Extranet","Custom","Internet"}, Values{"Default","Intranet","Extranet","Custom","Internet"}] String Zone;
+    [Required] String Url;
+    [Write, EmbeddedInstance("MSFT_Credential")] String InstallAccount;
+};
```
Comment (camiloborges) [inline Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.schema.mof]: Wharf if you need to remove an url?

File: Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.psm1 (hunk 1 above)
Comment (BrianFarnhill) [inline Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.psm1]: No, they are the unique keys for an alternate URL here, A different zone and web app would represent a different instance of this resource

File: Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.schema.mof (hunk 2 above)
Comment (BrianFarnhill) [inline Modules/xSharePoint/DSCResources/MSFT_xSPAlternateUrl/MSFT_xSPAlternateUrl.schema.mof]: bah ... yes I suppose you are right. I'll get on it!

**Pull request merged**